.github/workflows/e2e-test-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: e2e-test-ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - test/e2e-sanity-case
+jobs:
+  e2e-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install minikube
+        uses: abstractj/setup-minikube@issue-12
+      - name: Output cluster info
+        run: kubectl cluster-info
+      - name: Add images
+        run: |
+          minikube cache add apache/apisix:latest -v 7 --alsologtostderr
+          minikube cache add bitnami/etcd:3.4.14-debian-10-r0 -v 7 --alsologtostderr
+          minikube cache add kennethreitz/httpbin -v 7 --alsologtostderr
+          minikube cache add viewking/apisix-ingress-controller:dev -v 7 --alsologtostderr
```

Comment:
Why adding `viewking/apisix-ingress-controller:dev` cache ?
Do we need to deploy `apisix-ingress-controller`in minikube ?

Reply:
Oops, it seems i made a mistake here, we should build ingress-controller image each time.

Reply:
Fixed.